test/spec/old_import.wast

```diff
@@ -126,16 +126,6 @@
 (assert_trap (invoke "call" (i32.const 3)) "uninitialized element")
 (assert_trap (invoke "call" (i32.const 100)) "undefined element")
 
-
-(assert_invalid
-  (module (import "" "" (table 10 funcref)) (import "" "" (table 10 funcref)))
-  "multiple tables"
-)
-(assert_invalid
-  (module (import "" "" (table 10 funcref)) (table 10 funcref))
-  "multiple tables"
-)
```

Comment:
What happened here?

Reply:
We of course have supported multiple tables for a long time, and this test should have been removed when that support was introduced. But it was not noticed and continued failing successfully until now because of the empty import names.

Reply:
Are empty import names not valid? I thought they were.

Reply:
They are valid, but the legacy text parser was incorrectly rejecting them, which caused this `assert_invalid` check to pass. Now that I've fixed the legacy parser to allow empty names, this test started failing.